osi_lane.proto

```diff
@@ -678,6 +691,122 @@ message Lane
             //
             optional Identifier successor_lane_id = 2;
         }
+        
+        // A lane's surfaceline (as a list of segments).
+        //
+        // \brief Surfacelines describe the height profile along the lane
+        // Multiple surfacelines can be used to create a certain road shape if needed
+
+        // The surfaceline describes a height profile along the lane on or at a certain offset to the lane boundary.
+        // \image html OSI_LaneBoundaries_And_SurfaceLines.svg "Surfacelines" width=500px
+        //
+        // \note 
+        // sl: surfaceline
+        // lb: lane boundary
+        //
+        // \image html strassenquerschnitt_surfacelines.png "Lane shape surfacelines" width=500px
+        // 
+        // \attention The points of all surfacelines must be set in the 
+        // same ordering (ascending or descending) as the points describing the 
+        // lane boundaries or lane. Example: If the points are deducted from 
+        // a map format, the order of points is recommended to be in line with the 
+        // road coordinate (e.g. s-coordinate in OpenDRIVE). 
+        //
+        // \attention The points describing the surfaceline might be set at
+        // arbitrary distances. This allowes to model certain height changes along the 
+        // lane like speed bumps etc. more detailed while still allow very few points 
+        // at parts of the lane where height does not change much.
+        //
+        // \note The surfaceline is the line that describes the height profile 
+        // along the lane.
+        // 
```

Comment:
I would drop those lines because it is mentioned twice already (in the brief section of this message and usage of the message):
```suggestion
```